Dockerfile

```
@@ -50,7 +50,7 @@ COPY base/overrides/condor_ce_jobmetrics /usr/share/condor-ce/condor_ce_jobmetri
 
 # Workaround BatchRuntime expresion bug (HTCONDOR-506)
 COPY base/overrides/HTCONDOR-506.evalset-batchruntime.patch /tmp
-RUN patch -d / -p0 < /tmp/HTCONDOR-506.evalset-batchruntime.patch
+RUN [[ $BASE_YUM_REPO == 'development' ]] || patch -d / -p0 < /tmp/HTCONDOR-506.evalset-batchruntime.patch
```

```suggestion
RUN \
  python3 -c \
         'import sys; ver, rel = sys.argv[1:]; \
          ver_list = [int(x) for x in ver.split(".")]; \
          sys.exit(1 if (ver_list, rel) >= ([5, 1, 0], "1.1") else 0)' \
     $(rpm -q --qf "%{VERSION}  %{RELEASE}" htcondor-ce) \
  || patch -d / -p0 < /tmp/HTCONDOR-506.evalset-batchruntime.patch
```

Let's do a version comparison instead of depending on the state of our repos.

---

(also technically `1 if expr else 0` isn't necessary since in Python `True == 1` and `False == 0` but it's more readable this way)

---

Sure